packages/website/theme.config.js

```diff
@@ -1,4 +1,86 @@
+        transform="translate(-197.5 -171.5)"
+        fill="currentColor"
+      />
+      <g id="Oval-2">
+        <path
+          id="Oval"
+          d="M528.8,509q-6.3,6-13.2,11.4a190,190,0,1,1,38-40.2"
+          transform="translate(-197.5 -171.5)"
+          fill="none"
+          stroke="#5fc3e7"
+          stroke-width="18"
```

Comment:
Oh and I just noticed there are a few warnings on the console, the dashed attributes should be converted to camel case.